liblangutil/EVMVersion.cpp

```diff
@@ -76,6 +76,9 @@ bool EVMVersion::hasOpcode(Instruction _opcode, std::optional<uint8_t> _eofVersi
 	case Instruction::EXTCODECOPY:
 	case Instruction::GAS:
 		return !_eofVersion.has_value();
+	// Instructions below available only in EOF
+	case Instruction::DATALOADN:
+		return _eofVersion.has_value() && *this >= prague();
```

Comment:
The EVM version check is actually redundant since we now have an assert against this above.

```suggestion
		return _eofVersion.has_value();
```